scheduler/src/main/java/io/camunda/zeebe/scheduler/ActorScheduler.java

```diff
@@ -98,19 +102,23 @@ public static ActorSchedulerBuilder newActorScheduler() {
   }
 
   public static class ActorSchedulerBuilder {
+
     private String schedulerName = "";
     private ActorClock actorClock;
     private int cpuBoundThreadsCount = Math.max(1, Runtime.getRuntime().availableProcessors() - 2);
     private ActorThreadGroup cpuBoundActorGroup;
     private int ioBoundThreadsCount = 2;
     private ActorThreadGroup ioBoundActorGroup;
-
     private ActorThreadFactory actorThreadFactory;
     private ActorExecutor actorExecutor;
-
     private ActorTimerQueue actorTimerQueue;
-
     private boolean enableMetrics = false;
+    private Supplier<IdleStrategy> idleStrategySupplier =
+        ActorSchedulerBuilder::defaultIdleStrategySupplier;
+
+    public static IdleStrategy defaultIdleStrategySupplier() {
+      return new BackoffIdleStrategy(100, 100, 1, TimeUnit.MILLISECONDS.toNanos(1));
```

Comment:
💭 Could we use here the defaults from the config?

Reply:
If we want to have defaults specified only in one place, then we need to flip it. The `dist` module has to pull defaults from the `scheduler` module, since the dependency can only go on one way :+1:

Reply:
Hey, so apparently `BackoffIdleStrategy` already declares some defaults:

```java
    /**
     * Default number of times the strategy will spin without work before going to next state.
     */
    public static final long DEFAULT_MAX_SPINS = 10L;

    /**
     * Default number of times the strategy will yield without work before going to next state.
     */
    public static final long DEFAULT_MAX_YIELDS = 5L;

    /**
     * Default minimum interval the strategy will park a thread.
     */
    public static final long DEFAULT_MIN_PARK_PERIOD_NS = 1000L;

    /**
     * Default maximum interval the strategy will park a thread.
     */
    public static final long DEFAULT_MAX_PARK_PERIOD_NS = 1_000_000L;
```

Do you know why we opted for different ones?

Reply:
I have NO idea